images/public-log-vendor-matcher/pg-init.d/lib-08-download-c-ip-int.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+## Set a timestamp to work with
+TIMESTAMP=$(date +%Y%m%d%H%M)
+echo "$TIMESTAMP" > /tmp/my-timestamp.txt
+## Dump the entire table to gcs
+bq extract \
+  --destination_format CSV \
+  "${GCP_BIGQUERY_DATASET}_${PIPELINE_DATE}.2a_ip_int" \
+  "gs://ii_bq_scratch_dump/2a_ip_inti-$TIMESTAMP-*.csv" > "${BQ_OUTPUT:-/dev/null}" 2>&1
```

Comment:
found another hardcode, and below

Reply:
This has been swapped out in https://github.com/kubernetes/k8s.io/commit/81f93b365f24ff638486c6cc2ae9b3c6d7e26e56